Assets/MixedRealityToolkit/Inspectors/Utilities/HandednessInspectorGUI.cs

```diff
@@ -0,0 +1,29 @@
+﻿
+using UnityEditor;
+using UnityEngine;
+
+public static class HandednessInspectorGUI
```

Comment:
Please add to the appropriate namespace -- MRTK classes should be in a mrtk namespace. Please see [this guideline](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#mrtk-namespace-rules) for more.

Reply:
> I don't feel a separate file is really needed in this case as it's only used by a single area, but abstraction can help the flow of code.
> 
> My recommendation would be to put the function inside the inspector class, but keep it as a separate function for clarity and not create a new class file.

Do you mean DrawControllerHandednessDropdown? It's used in two classes, ControllerPoseSynchronizerInspector and MixedRealityControllerVisualizationProfileInspector. If you want I create a seperate pull request that deals with this extraction